rotkehlchen/tests/api/test_airdrops.py

```diff
@@ -0,0 +1,97 @@
+from typing import TYPE_CHECKING
+
+import requests
+
+from rotkehlchen.tests.utils.api import api_url_for, assert_proper_response_with_result
+
+if TYPE_CHECKING:
+    from rotkehlchen.api.server import APIServer
+
+
+def test_airdrops_endpoint(rotkehlchen_api_server: 'APIServer') -> None:
+    """Test that the airdrops endpoint works properly"""
+    response = requests.get(api_url_for(rotkehlchen_api_server, 'airdropsmetadataresource'))
+
+    result = assert_proper_response_with_result(response)
+    assert result == [
+        {
+            'identifier': 'uniswap',
```

Comment:
same comment, instead of pasting the whole list can you just generate it from the original one?